### PR TITLE
Layout create branch

### PR DESCRIPTION
**Pull request by KirtMorgan**

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Our Amazing Flask</title>
```
Comment (KirtMorgan) [inline templates/layout.html]: Change this to The Food Mood Blog

File: templates/story.html [hunk 2]
```diff
@@ -1,41 +1,28 @@
 <!DOCTYPE html>
```
Comment (KirtMorgan) [inline templates/story.html]: This is no longer needed, because the layout will provide this

File: templates/story.html [hunk 3]
```diff
@@ -1,41 +1,28 @@
...
+    <p>So here we are, 3 world renowned chefs who have come together from different corners of the world for one purpose... </p>
+
+    <h4 style="text-align: center; color: red">To cook DELICIOUS FOOD that will put you in the MOOD
+        and DOCUMENT it so that you can cook them using our recipes too!</h4>
+
+    <p>Enjoy our blog!</p>
+    <h2>🍝Bon appetite!🍝</h2>
+{% endblock %}
+
+</html>
```
Comment (KirtMorgan) [inline templates/story.html]: Remove this as well

File: templates/layout.html (hunk 1 above)
Comment (TahsinKhan7) [reply templates/layout.html]: ok ill do this

**Pull request merged**